cluster/addons/cluster-monitoring/influxdb/influxdb-grafana-petset.yaml

```diff
@@ -65,8 +65,12 @@ spec:
             mountPath: /var
       volumes:
       - name: influxdb-persistent-storage
+{% if grains['cloud'] == 'gce' %}
```

Comment:
This file doesn't have any jinja templating in it right now, and I think we are striving to keep the `cluster/addons` directory templating free. 

@mikedanese